test/createAzureResource.test.ts

```diff
@@ -25,6 +25,7 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
     const resourceName1: string = getRandomHexString().toLowerCase();
     // Get the *.code-workspace workspace file path
     const projectPath: string = getTestRootFolder();
+    const applicationSettings: string[] = ['FUNCTION_EXTENSION_URL', "https://github.com/microsoft/vscode-azurefunctions"];
```

Comment:
I don't think you need to define an applicationSettings array here.  I would just define a `const key: string` and a `const value: string`.

This isn't being used by any other tests (yet) so it doesn't need to be up here.  Though I assume that you're going to use these to test edit and delete next though?